lib/purposes/ControllerProofPurpose.js

```diff
@@ -66,7 +67,6 @@ module.exports = class ControllerProofPurpose extends ProofPurpose {
           }
           controllerId = owner;
```

Comment:
We need a similar fix here with owner.

Reply:
Done: https://github.com/digitalbazaar/jsonld-signatures/pull/91/commits/81d55746c846c55d9b96041f8489f30d15c2b570